modules/social_features/social_profile/modules/social_profile_fields/config/modify/social_profile_fields.taxonomy_permissions.yml

```diff
@@ -0,0 +1,23 @@
+dependencies:
+  module:
+    - social_registration_fields
+    - taxonomy_access_fix
+items:
+  user.role.anonymous:
+    expected_config:
+      permissions: { }
+    update_actions:
+      add:
+        permissions:
+          - 'view terms in nationality'
+          - 'view term names in nationality'
+          - 'select terms in nationality'
+  user.role.authenticated:
+    expected_config:
+      permissions: { }
+    update_actions:
+      add:
+        permissions:
```

Comment:
I think this one is actually not depending on `social_registration_fields` as authenticated user (and verified and such) you should be able to view and select these, as they should be able to edit their profile.

Reply:
The edit profile page is working without this permission, but the registration page using registration fields modules are looking for the new permission.

Reply:
I see this has been split up now where the anonymous yml is with the registration fields module, and the authenticated+ yml is only depending on taxonomy access fix.

So putting it as resolved.